tasks/interactive-bots.md

```diff
@@ -110,8 +110,8 @@ doing this setup process. Add your notes as a `notes.md` file to
 * Create a commit with the screenshots and notes, with commit message
 `interactive bots: Run the followup bot for <username>.`.
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the followup bot for <username>`.
+* Create a pull request in the (`zulip/zulip-gci`)[https://github.com/zulip/zulip-gci] repository, with title
```

Comment:
I don't think links are necessary here - we would have to change it in all the tasks to be consistent and I'd think a GCI contestant knows which repository we mean from the beginner tasks.